medical_prescription/tests/test_medical_prescription_order.py

```diff
@@ -9,24 +9,27 @@ class TestMedicalPrescriptionOrder(TransactionCase):
 
     def setUp(self):
         super(TestMedicalPrescriptionOrder, self).setUp()
+        self.rx_1 = self.env.ref(
+            'medical_prescription.medical_prescription_prescription_order_1'
+        )
+        self.rx_6 = self.env.ref(
+            'medical_prescription.medical_prescription_prescription_order_6'
+        )
 
-        self.line_model = self.env['medical.prescription.order.line']
-        self.line = self.env.ref(
-            'medical_prescription.'
-            'medical_prescription_order_line_patient_1_order_1_line_1'
+    def test_sequence_for_name(self):
+        """ Test name created if there is none """
+        self.assertTrue(
+            self.rx_1.name,
         )
 
-    def test_active_true(self):
-        """ It should set active to true when rx line is active """
+    def test_compute_active(self):
+        """ Test active is True if rx line_ids present """
         self.assertTrue(
-            self.line.prescription_order_id.active
+            self.rx_1.active,
         )
 
-    def test_active_false(self):
-        """ It should set active to true when rx line is active """
-        order = self.line.prescription_order_id
-        for line in order.prescription_order_line_ids:
-            line.active = False
+    def test_compute_active_no_line_ids(self):
+        """ Test active is False if no rx line_ids """
         self.assertFalse(
-            self.line.prescription_order_id.active
+            self.rx_6.active,
```

Comment:
`medical_prescription_prescription_order_6` has no `prescription_order_line_ids` which according to medical_prescription/models/medical_prescription_order.py:_compute_active results in the RX being `active=True`